### PR TITLE
Add Docker build and run scripts

### PR DESCRIPTION
**Pull request by ProchaLu**
## Description
This PR adds three new scripts to the package.json file:
- `docker-build`: Build a Docker image 
- `docker-run`: Run the Docker image
- `docker-build-run`: Build a Docker image and immediately run the Docker image

This makes building and running a Docker image easier, simplifying the deployment process.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,10 @@
   "scripts": {
     "analyze": "size-limit --why",
     "build": "tsdx build --format esm --target node",
-    "build-docker-ts": "tsc --project docker/tsconfig.build.json",
+    "docker-build": "pnpm docker-build-ts && docker build -t preflight .",
```
Comment (karlhorky) [inline package.json:26]:
```suggestion
    "docker-build": "docker build -t preflight .",
```

File: package.json [hunk 2]
```diff
@@ -23,7 +23,10 @@
...
+    "docker-build-run": "pnpm docker-build-ts && docker build -t preflight . && docker run preflight",
```
Comment (karlhorky) [inline package.json:27]:
```suggestion
    "docker-build-run": "pnpm docker-build-ts && pnpm docker-build && pnpm docker-run",
```

File: package.json [hunk 3]
```diff
@@ -23,7 +23,10 @@
   "scripts": {
     "analyze": "size-limit --why",
     "build": "tsdx build --format esm --target node",
-    "build-docker-ts": "tsc --project docker/tsconfig.build.json",
+    "docker-build": "docker build -t preflight .",
```
Comment (karlhorky) [inline package.json:26]: is there a long version of this flag? (one that starts with `--`)

Comment (ProchaLu) [reply package.json]: Yes `--tag`
[05cb8a5](https://github.com/upleveled/preflight/pull/393/commits/05cb8a5148fa01867bd7a61f461587f0eb2eb193)

**Review by karlhorky — APPROVED** [review]

**Pull request merged**